app/src/main/java/org/gdg/frisbee/android/adapter/ChapterAdapter.java

```diff
@@ -76,4 +76,14 @@ public long getItemId(int position) {
         else
             return 0;
     }
+
+    public Chapter findById(final String chapterId) {
+        for (int i = 0; i < getCount(); i++) {
```

Comment:
The list is not exposed to the outside, so I don't see a possiblity here. getAll() iterates as well through all elements.